### ISSUE TITLE
Create a Quarkus GraphQL example app

### ISSUE THREAD

**Issue by beikov** [Blazebit/blaze-persistence#1116]
Quarkus has a GraphQL extension and I'd like to have a sample application and maybe a blog post about the use of GraphQL with Entity-Views. I hope that our existing integration works. If not, it should be adapted.

**Comment by minhdtb** [comment]
@beikov yeah, I really need Quarkus graphql example by using smallrye-graphql. Hope you add it soon
